core/src/packet/packet_ms.cpp

```diff
@@ -363,12 +363,13 @@ AOPacket *PacketMS::validateIcPacket(AOClient &client) const
     }
 
     // Testimony playback
+    QString client_name = client.m_ooc_name == "" ? client.m_current_char : client.m_ooc_name;
```

Comment:
Please write this as an if/else for clarity

Reply:
you're the boss